tar.go

```diff
@@ -29,11 +30,13 @@ func NewTarStreamer(r io.Reader, keywords []string) Streamer {
 		tarReader:  tar.NewReader(pR),
 		keywords:   keywords,
 	}
-	go ts.readHeaders() // I don't like this
+
+	go ts.readHeaders()
```

Comment:
For your second question, if both `tar_time` and `time` are defined, the `tar_time` evaluation should be done; basically, `tar_time` "wins" against `time`, since it is a more general evaluation of a file's mtime. Can you confirm this, @vbatts?